packages/core/admin/admin/src/content-manager/components/DynamicTable/index.js

```diff
@@ -89,17 +90,23 @@ const DynamicTable = ({
 
   return (
     <Table
-      components={{ ConfirmDialogDelete, ConfirmDialogDeleteAll }}
+      components={{ ConfirmDialogDelete }}
```

Comment:
Can't we remove this and let `bulkActionsBar` handle `ConfirmDialogDelete` as well? This way components is not needed when `bulkActionsBar` is passed